src/config/transform.rs

```diff
@@ -163,7 +163,7 @@ impl TransformContext {
         }
     }
 
-    #[cfg(any(test, feature = "test"))]
+    #[cfg(test)]
```

Comment:
This might break other users who use the `test` feature.

Reply:
AFAICT that is only ever called from `src/transforms/remap.rs` in its test code, so it being unreachable unused code when built with just the `test` feature tracks.

Reply:
Interesting, `TransformContext` is exposed though. This only has one usage and all fields are `pub` already, maybe it can be deleted/refactored.